test.py

```diff
@@ -0,0 +1,7 @@
+import gym
+import gym_frc
+
+env = gym.make('Graph-v2019')
```

Comment:
Should specify frc, for now, I feel the best idea would likely be `FrcGraph2019-v1`, the `v` should specify an actual version if we update old versions in the future, where as 2019 is the game year. This would best follow gym convention